docs/api/write/insertClassSectionEvents.md

```diff
@@ -111,14 +111,14 @@ Properties marked with the quote "consult [#Constants](#constants) for the avail
     - e.g. `[ "A.2.1", "G.0.21" ]`
     - unique items (e.g. `[ "A.2", "A.2" ]` is not allowed)
     - this value does not have a strict format, since other languages may require the use of special characters for identifying locations (e.g. Mandarin Chinese)
-* `beginTime`
-    - *time* type
-    - `hh:mm` format 
-    - e.g. `14:00`
-* `duration`
-    - *time* type
-    - `hh:mm` format 
-    - e.g. `01:30`
+* `startDate`
+    - *datetime* type
+    - `yyyy-mm-ddThh:mm` format 
+    - e.g. `2020-01-01T10:10`
+* `endDate`
+    - *datetime* type
+    - `yyyy-mm-ddThh:mm` format 
+    - e.g. `2020-01-01T10:10`
```

Comment:
Why does #213 exist if the type here is `datetime`?

Reply:
Is it for non recurrent events?

Reply:
For punctual events, which ommit the weekday property. the recurrent events will have this portion ignored and rather use the calendar term's span.

Reply:
Seems like there should be two types of objects then no?

Reply:
Does it make sense to have one endpoint for recurring events and one for non recurring events? Where the request body differs between them.

Reply:
@joao-pereira5 Is it possible to specify, in the schema, that `startDate` and `endDate` are of type `time` when `weekday` is present and `datetime` when it is not?

Reply:
With regex patterns. Absolutely